release_notes/ocp_3_9_release_notes.adoc

```diff
@@ -0,0 +1,434 @@
+and others are only running `docker`.
+
+See the link:http://cri-o.io/[upstream documentation] for more information on
+CRI-O.
+
+[[ocp-39-]]
+====
+
+
+[[ocp-39-]]
+====
+
+[[ocp-39-security]]
+=== Security
+
+[[ocp-39]]
+====
+
+[[ocp-39-storage]]
+=== Storage
+
+[[ocp-39-]]
+====
+
+[[ocp-39-scale]]
+=== Scale
+
+[[ocp-39-scale-cluster-limits]]
+==== Cluster Limits
+
+Updated guidance around
+xref:../scaling_performance/cluster_limits.adoc#scaling-performance-cluster-limits[Cluster
+Limits] for {product-title} 3.9 is now available.
+
+[[ocp-39-device-plugin]]
+==== Device Plug-in
+
+[[ocp-39-CPU-manager]]
+==== CPU Manager
+
+[[ocp-39-device-manager]]
+==== Device Manager
+
+[[ocp-39-hugepages]]
+==== HugePages
+
+
+
+
+[[ocp-39-networking]]
+=== Networking
+
+
+[[ocp-39-master]]
+=== Master
+
+[[ocp-39-]]
+====
+
+
+[[ocp-39-installation]]
+=== Installation
+
+
+
+[[ocp-39-diagnostics]]
+=== Diagnostics
+
+[[ocp-39-metrics-and-logging]]
+=== Metrics and Logging
+
+[[ocp-39-]]
+====
+
+[[ocp-39-developer-experience]]
+=== Developer Experience
+
+[[ocp-39-cli-plug-ins]]
+==== CLI Plug-ins
+
+CLI plug-ins are now fully supported.
+
+Usually called _plug-ins_ or _binary extensions_, this feature allows you to
+extend the default set of `oc` commands available and, therefore, allows you to
+perform new tasks.
+
+See xref:../cli_reference/extend_cli.adoc#cli-reference-extend-cli[Extending the
+CLI] for information on how to install and write extensions for the CLI.
+
+[[ocp-39-]]
+==== d
+
+
+
+[[ocp-39-web-console]]
+=== Web Console
+
+
+[[ocp-39-notable-technical-changes]]
+== Notable Technical Changes
+
+{product-title} 3.9 introduces the following notable technical changes.
+
+[discrete]
+[[several-oc-secrets-subcommands-now-deprecated]]
+=== Several oc secrets Subcommands Now Deprecated
+
+{product-title} 3.9 deprecates the following `oc secrets` subcommands in favor
+of `oc create secret`:
+
+* `new`
+* `new-basicauth`
+* `new-dockercfg`
+* `new-sshauth`
+
+
+[discrete]
+[[updated-default-installer-values]]
+=== Updated Default Values for template_service_broker_prefix and template_service_broker_image_name in the Installer
+Default values for `template_service_broker_prefix` and `template_service_broker_image_name` in installer have been updated to be consistent with other settings.
+
+Previous values are:
+
+    template_service_broker_prefix="registry.example.com/"
+    template_service_broker_image_name="origin-template-service-broker" or template_service_broker_image_name="ose-template-service-broker"
+
+New values are:
+    template_service_broker_prefix="registry.example.com/ose-" or template_service_broker_prefix="registry.example.com/origin-"
+    template_service_broker_image_name="template-service-broker"
+
+
+[discrete]
+[[removed-become-no-instances]]
+=== Removed several instances of 'become: no' on certain tasks and playbooks inside of openshift-anisble
```

Comment:
It's not clear to me why this is important enough for a release note, it seems VERY low level and something that we as an installer should be taking care of for users.